src/components/Project/index.js

```diff
@@ -11,7 +11,7 @@ const Project = ({ title, info, symbol }) => {
           <S.Project_info>{info}</S.Project_info>
         </S.Text_content>
         <S.Icon_layout>
-          <S.Icon src={require(`../../../assets/${symbol}`)} />
+          <S.Icon src={require(`../../assets/${symbol}`)} />
```

Comment:
ES6 문법인 import로 고치면 좋을 것 같아욧

Reply:
앗 해당 symbol 확인을 못했네요 음~ 저라면

```suggestion
          <S.Icon src={`../../assets/${symbol}`} />
```

이렇게 할 것 같아요 require은 불필요해보여요!